src/ledger/ledger.test.js

```diff
@@ -214,6 +214,73 @@ describe("ledger/ledger", () => {
       });
     });
 
+    describe("changeIdentitySubtype", () => {
+      it("works", () => {
```

Comment:
something more descriptive of the functionality would be better

```suggestion
      it("changes a USER identity type into a PROJECT identity type", () => {
```